src/forwarders.js

```diff
@@ -572,6 +543,50 @@ export default function Forwarders(mpInstance, kitBlocker) {
         }
     };
 
+    // kits can be included via mParticle UI, or vi
+    this.configureSideloadedKit = function(sideloadedKit) {
+        mpInstance._Store.configuredForwarders.push(
+            this.returnConfiguredKit(sideloadedKit)
+        );
+    };
+
+    this.returnConfiguredKit = function(forwarder, config = {}) {
```

Comment:
I've been thinking about this for a while, but I wonder if we should create a Forwarder class or factory that takes a config and constructor, and returns an instance of a forwarder.

Reply:
https://go.mparticle.com/work/SQDSDKS-5161